lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -146,6 +146,19 @@ <h3 class="hd hd-3" id="header-${ section_data['section_key'] }">${ section_data
   </div>
 </div>
 
+<%static:optional_include_mako file="google_analytics.html" is_theming_enabled="True" />
+<% ga_acct = static.get_value("GOOGLE_ANALYTICS_ACCOUNT", settings.GOOGLE_ANALYTICS_ACCOUNT) %>
```

Comment:
Ideally, I think the only integration we should have in this file is a single, optional include, to be loaded from our theme.

Also we should _not_ use an inline script, but should instead use `<script src="PATH_TO_JS_FILE" ...>`;
this allows the file/contents to be cached, instead of resent on each pageload.

Reply:
(the optional include should be for an HTML file, something like `instructor_link_analytics.html` or something like that...)

Reply:
It should be something generically named, like `instructor_dashboard_bottom.html` or some such?